codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/generators/client/FluentClientGenerator.kt

```diff
@@ -89,139 +93,147 @@ class FluentClientGenerator(
+                    val operationOk = symbolProvider.toSymbol(output)
+                    val operationErr = symbolProvider.symbolForOperationError(operation)
+
+                    val inputFieldsBody = generateOperationShapeDocs(
+                        this,
+                        codegenContext,
+                        symbolProvider,
+                        operation,
+                        model,
+                    ).joinToString("\n") { "///   - $it" }
+
+                    val inputFieldsHead = if (inputFieldsBody.isNotEmpty()) {
+                        "The fluent builder is configurable:"
+                    } else {
+                        "The fluent builder takes no input, just [`send`]($fullPath::send) it."
+                    }
 
-                // Write a deprecation notice if this operation is deprecated.
-                writer.deprecatedShape(operation)
+                    val outputFieldsBody =
+                        generateShapeMemberDocs(this, symbolProvider, output, model).joinToString("\n") {
+                            "///   - $it"
+                        }
 
-                writer.rustTemplate(
-                    """
-                    pub fn #{fnName}(&self) -> #{FluentBuilder}${generics.inst} {
-                        #{FluentBuilder}::new(self.handle.clone())
+                    var outputFieldsHead = "On success, responds with [`${operationOk.name}`]($operationOk)"
+                    if (outputFieldsBody.isNotEmpty()) {
+                        outputFieldsHead += " with field(s):"
                     }
-                    """,
-                    "fnName" to writable { rust(clientOperationFnName(operation, symbolProvider)) },
-                    "FluentBuilder" to operation.fluentBuilderType(codegenContext, symbolProvider),
-                )
+
+                    rustTemplate(
+                        """
+                        /// Constructs a fluent builder for the [`$name`]($fullPath) operation.$maybePaginated
+                        ///
+                        /// - $inputFieldsHead
+                        $inputFieldsBody
+                        /// - $outputFieldsHead
+                        $outputFieldsBody
+                        /// - On failure, responds with [`SdkError<${operationErr.name}>`]($operationErr)
+                        """,
+                    )
+
+                    // Write a deprecation notice if this operation is deprecated.
+                    deprecatedShape(operation)
+
+                    rustTemplate(
+                        """
+                        pub fn $fnName(&self) -> #{FluentBuilder}${generics.inst} {
+                            #{FluentBuilder}::new(self.handle.clone())
+                        }
+                        """,
+                        "FluentBuilder" to operation.fluentBuilderType(codegenContext, symbolProvider),
+                    )
+                }
             }
         }
     }
```

Comment:
You didn't touch this in your PR, but on line 249 below, where it says:
```kt
rust(
    """
    /// Fluent builder constructing a request to `${operationSymbol.name}`.
    ///
    """,
)
```

We have an opportunity to use the `docs` writer and turn the `${operationSymbol.name}` into a link to more docs.